scrapers/utils/actions.py

```diff
@@ -26,13 +26,13 @@ def __new__(
             regexes = (regexes,)
         compiled_regexes = []
         # pre-compile any string regexes
-        for regex in regexes:
```

Comment:
This part's ugly, but before the `regex` variable was changing type during the for loop, which could get confusing.